code/game/objects/items/holomenu.dm

```diff
@@ -74,7 +74,7 @@
 	return ..()
 
 /obj/item/holomenu/examine(mob/user, distance, is_adjacent)
-	if(anchored && length(menu_text) && is_adjacent)
+	if(anchored && length(menu_text))
```

Comment:
You should check for a reasonable distance, to avoid examining menus across the map.